app/src/main/java/org/oppia/android/app/testing/DrawableBindingAdaptersTestActivity.kt

```diff
@@ -0,0 +1,15 @@
+package org.oppia.android.app.testing
+
+import android.os.Bundle
+import org.oppia.android.R
+import org.oppia.android.app.activity.ActivityComponentImpl
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+
+/** Test activity for ImageViewBindingAdapters. */
```

Comment:
Is this documentation correct?

Reply:
Oh! My mistake as I forgot to do these changes. I will rectify them and others.